graphdatascience/session/aura_api.py

```diff
@@ -71,21 +71,23 @@ def create_session(self, name: str, dbid: str, pwd: str, memory: SessionMemoryVa
 
         return SessionDetails.fromJson(response.json()["data"])
 
-    def list_session(self, session_id: str, dbid: str) -> Optional[SessionDetails]:
-        response = self._request_session.get(
-            f"{self._base_uri}/v1beta5/data-science/sessions/{session_id}?instanceId={dbid}",
-        )
-
-        if response.status_code == 404:
-            return None
+    def list_session(self, session_id: str) -> Optional[SessionDetails]:
```

Comment:
The is not super related to this PR, but I think list_session is a bad name, listing imho returns a list not a single item.